tests/manage/cluster/nodes/test_nodes_restart.py

```diff
@@ -34,4 +35,61 @@ def test_nodes_restart_aws(self, resources, instances, aws_obj, force):
         aws_obj.restart_ec2_instances(instances=instances, wait=True, force=force)
         self.validate_cluster(resources=resources, nodes=list(instances.values()))
 
+    @pytest.mark.parametrize(
+        argnames=["wait_for_not_ready_nodes"],
+        argvalues=[
+            pytest.param(*[True], marks=pytest.mark.polarion_id("OCS-1138")),
+            pytest.param(*[False], marks=pytest.mark.polarion_id("OCS-1139"))
+        ]
+    )
+    def test_pv_provisioning_under_degraded_state(
+        self, resources, instances, aws_obj, wait_for_not_ready_nodes
+    ):
+        """
+        Test PV provisioning under degraded state
+
+        OCS-1138:
+        - Stop 1 ec2 instance worker node
+        - Wait for the worker node to reach status 'NotReady'
+        - Validate cluster functionality, without checking cluster and Ceph
+          health
+          by creating resources and running IO
+        - Start the worker node ec2 instance
+        - Check cluster and Ceph health
+
+        OCS-1139:
+        - Stop 1 ec2 instance worker node
+        - Without waiting for the worker node to reach status 'NotReady' -
+        - Validate cluster functionality, without checking cluster and Ceph
+          health
+          by creating resources and running IO
+        - Start the worker node ec2 instance
+        - Check cluster and Ceph health
+
+        """
+        # TODO: The number of nodes to stopped should be dynamic, according
+        #  to the minimum number of nodes needed for a functioning cluster
+        workers = node.get_typed_nodes(num_of_nodes=1)
+
+        worker_names = [worker.get('metadata').get('name') for worker in workers]
+        instances = aws.get_instances_ids_and_names(workers)
+
+        # Stopping the nodes
+        aws_obj.stop_ec2_instances(instances=instances, wait=True)
+
+        if wait_for_not_ready_nodes:
+            # Waiting for the stopped nodes to reach status 'Not Ready'
+            assert node.wait_for_nodes_status(
+                node_names=worker_names, status=constants.NODE_NOT_READY, timeout=600
+            ), f"Not all nodes reached status {constants.NODE_NOT_READY}"
```

Comment:
@ebenahar not a priority ... but instead of generic assert statement(which may also seem to mean **all** nodes - master+ worker), shall we refine it to mean the specific worker node ( since we are only powering off worker node? )

Reply:
@nehaberry this assert message is redundant actually, as wait_for_nodes_status() will log with error the nodes that failed to reach the status. 
In the new commit, I'm going to remove the call to this function completely